_docs_v7/Slope-Limiters-and-Shock-Resolution.md

```diff
@@ -0,0 +1,205 @@
+More possible applications of limiters are listed below.
+
+
+### Slope Limiter Fields
+
+| Configuration Field | Description | Notes |
+| --- | --- | --- |
+| `SLOPE_LIMITER_FLOW` | Flow equations | Need `MUSCL_FLOW = YES` |
+| `SLOPE_LIMITER_TURB` | Turbulence equations | Need `MUSCL_TURB = YES` |
+| `SLOPE_LIMITER_SPECIES` | Species evolution equations | Need `MUSCL_SPECIES = YES` |
+| `SLOPE_LIMITER_ADJFLOW` | Adjoint flow equations | Need `MUSCL_ADJFLOW = YES` |
+| `SLOPE_LIMITER_ADJTURB` | Adjoint turbulence equations | Need `MUSCL_ADJTURB = YES` |
+
+
+<!-- 
+CLimiterDetails.hpp:
+computeLimiters.hpp uses computeLimiters_impl.hpp and really just sets the limiter
+computeLimiters_impl.hpp passes computation of field values (cell averages), gradients, and  to the limiter
+also check out
+computeGradientsGreenGauss.hpp (and the least squares one)
+ -->
+
+The `SLOPE_LIMITER_` options above may each be changed to use different limiters, which are listed and explained below.
+
+**Note:** the Discontinuous-Galerkin methods (DG) / Higher-order methods (HOM) do not use limiters.
+
+
+
+### Available Limiters
+
+| Type | Description | Notes |
+| --- | --- | --- |
+| `NONE`                  | No limiter                                      |  |
+| `BARTH_JESPERSEN`       | Barth-Jespersen                                 |  |
+| `VENKATAKRISHNAN`       | Venkatakrishnan                                 |  |
+| `VENKATAKRISHNAN_WANG`  | Venkatakrishnan-Wang                            |  |
+| `SHARP_EDGES`           | Venkatakrishnan with sharp-edge modification    | This limiter should not be used for flow solvers |
+| `WALL_DISTANCE`         | Venkatakrishnan with wall distance modification | This limiter should not be used for flow solvers |
+| `VAN_ALBADA_EDGE`       | Van Albada (edge formulation)                   | This limiter has been observed to not output limiter values when called using VOLUME_OUTPUT |
+
+<!-- ??? Van Albada. It's possible that we actually have this backward and Van Albada is the only limiter that works for other solvers??? -->
+<!-- We're currently investigating if Barth-Jespersen is Venkat with K=0 -->
+
+The default limiter is `VENKATAKRISHNAN`.
+
+### Limiter Parameters and Further Details
+
+The `VENKAT_LIMITER_COEFF` parameter is generally a small constant, defaulting to $$0.05$$, but its specific definition depends on the limiter being used.
+This is different than the small constant used to prevent division by zero, which is used in all these limiters.
+<!-- ^ I'm refering to: `static Type epsilon() {return std::numeric_limits<passivedouble>::epsilon();}` in CLimiterDetails.hpp -->
+
+For the `VENKATAKRISHNAN`, `SHARP_EDGES`, and `WALL_DISTANCE` limiters, the `VENKAT_LIMITER_COEFF` parameter refers to $$K$$ in $$\epsilon^2=\left(K\bar{\Delta} \right)^3$$, where $$\bar{\Delta}$$ is an average grid size.
```

Comment:
This is another inconsistency we have with the literature, I think in the original Venkatakrishan paper a local grid size is used (based on the volume of the CV).

Reply:
I was wrong, the original reference uses an average size for all Cvs, it was in a different paper that I saw a local reference for each Cv https://arc.aiaa.org/doi/abs/10.2514/6.2022-1374